docs/UserGuide.md

```diff
@@ -9,331 +9,424 @@ Interface** (CLI) while still providing an easy way to visualize all events thro
 
-Parameters:  
-`n/`: Name or description of event  
+-------------------------------------
+
+<h2 id="quick-start">Quick Start</h2>
+
+1. Ensure you have Java `11` or above installed in your Computer.
+2. Download the latest `Ez-Schedule.jar` from [here](https://github.com/AY2223S2-CS2103-W17-3/tp/releases).
+3. Copy the file to the folder you want to use as the *home folder* for your _Ez-Schedule_.
+4. Open a command line terminal, `cd` into the folder you put the jar file in
+5. Use the `java -jar Ez-Schedule.jar` command to run the application.
+
+[[Back to top](#table-of-contents)]
+
+-------------------------------
+
+<h2 id="commands">Commands</h2>
+
+<h3 id="add">Add a new event `add`</h3>
+
+Adds an event into the scheduler.
+
+Format: `add n/<event> d/<date> s/<start time> e/<end time>`  
+Example: `add n/tennis d/2023-04-20 s/15:00 e/17:00`
+
+Parameters:
+`n/`: Name of the event  
 `d/`: Date of the event  
 `s/`: Start time of the event  
 `e/`: End time of the event
 
-<div markdown="span" class="alert alert-warning">
+<div markdown="span" class="alert alert-info">
 :information_source: **Note:**  
-Name/Description is alphanumeric only.  
-:warning: Start time should come before end time.  
-Adding of events that overlaps in time is not allowed.
+Name is alphanumeric only.  
+Date should follow a `YYYY-MM-DD` format.  
+Time should be in 24-hour `HH:mm` format, ranging from `00:00` to `23:59`. Leading zero are required.
 </div>
 
-Example: `add n/booked tennis court d/2023-02-01 s/15:00 e/17:00`
-
-[[Back to top](#index)]
+<div markdown="span" class="alert alert-warning">
+:warning: **Warning:**  
+Start time should come before.  
+Adding of events that overlaps in time is not allowed.
+</div>
 
+[[Back to top](#table-of-contents)]
 
-<h3 id="recur">Repeat Existing Event</h3>
+<h3 id="recur">Repeat an existing event `recur`</h3>
 
-Recurs an event from the Scheduler until an end date.
+Repeats an event from the scheduler until the specified end date.
 
-Format: `recur <index> d/<end date> every/<recur factor>`
+Format: `recur <index> d/<end date> every/<recur factor>`   
+Example: `recur 1 d/2024-01-01 every/month`
 
 Parameters:  
-`d/`: Ending date of recurrence for the event in `YYYY-MM-DD` format  
-`every/`: Recur factor to recur the event, given by `day/week/month`
+`d/`: Ending date of recurrence for the event  
+`every/`: Recurring factor to recur the event
+
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:**.  
+Date should follow a `YYYY-MM-DD` format.  
+Recurring factor is given by `day/week/month`.
+</div>
 
 <div markdown="span" class="alert alert-warning">
-:information_source: **Note:**  
+:warning: **Warning:**  
 Event must first exist in the scheduler.  
-Recurring of events not existing in scheduler is not allowed.  
 End date should come after current event date.  
+Recurring of events not existing in scheduler is not allowed.
 </div>
 
-Example: `recur 1 d/2024-01-01 every/month`
```

Comment:
I feel that it would be more intuitive for parameters should follow directly below the format, instead of example.